components/mpas-ocean/src/shared/mpas_ocn_frazil_forcing.F

```diff
@@ -600,8 +606,8 @@ subroutine ocn_frazil_forcing_build_arrays(domain, meshPool, forcingPool, stateP
                                                  / dt
 
               ! keep track of new frazil ice
-              sumNewThicknessWeightedSaltContent = sumNewThicknessWeightedSaltContent - meltedThicknessWeightedSaltContent
-              sumNewFrazilIceThickness = sumNewFrazilIceThickness - meltedFrazilIceThickness
+              sumNewThicknessWeightedSaltContent = max(0.0_RKIND,sumNewThicknessWeightedSaltContent - meltedThicknessWeightedSaltContent)
+              sumNewFrazilIceThickness = max(0.0_RKIND,sumNewFrazilIceThickness - meltedFrazilIceThickness)
```

Comment:
This change shouldn't be necessary because `meltedFrazilIceThickness` is already limited to `sumNewFrazilIceThickness` and consequently `meltedThicknessWeightedSaltContent` is limited to `sumNewThicknessWeightedSaltContent`

Reply:
@cbegeman : It's there to prevent negatives from roundoff errors.

Reply:
Hmm, wouldn't have thought that would happen based on the operations above but better to be safe.